src/CubedSpheres/cubed_sphere_halo_filling.jl

```diff
@@ -11,10 +11,10 @@ fill_north_halo!(c, bc::CubedSphereExchangeBC, args...; kwargs...) = nothing
 
 function fill_halo_regions!(field::AbstractCubedSphereField, arch, args...; kwargs...)
 
-    for face_field in faces(field)
-        # Fill the top and bottom halos the usual way.
-        fill_halo_regions!(face_field, arch, args...; kwargs...)
+    # Fill the top and bottom halos the usual way.
+    #fill_halo_regions!(field, arch, args...; kwargs...)
```

Comment:
Are bottom and top halos filled elsewhere now or do we still need this?

Reply:
Hmm, I'll put it back in.